assets/components/threeSubscriptions/threeSubscriptions.jsx

```diff
@@ -8,73 +8,126 @@ import PageSection from 'components/pageSection/pageSection';
 import SubscriptionBundle from 'components/subscriptionBundle/subscriptionBundle';
 
 
+// ----- Setup ----- //
+
+const gridImageProperties = {
+  srcSizes: [825, 500, 140],
```

Comment:
I don't know if I am reading this grigImagesProperties correctly, but I think here we requesting an image of 825px but according to the line 15 we use at most 205?. Is this correct?

Reply:
Those are the available sizes of the image on the Grid. With responsive images you tell the browser what sizes of image are available on the server, and you give it the information about how big the slot in the page is going to be at different breakpoints. Then you let it figure out what size of image it needs to request for the current viewport size, screen resolution etc.

In this case the Grid offers three versions of this image at different sizes: 825, 500 and 140px. We've told the browser how big the slots are at different breakpoints, sometimes it's 165px, sometimes it's 205px etc. If the browser is on a breakpoint where it needs to fill a slot of 205px it will usually grab the 500px image, not the 825px one.

However, this gets more complicated when [device pixel ratios](https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio) come into play, because CSS pixels don't line up with device pixels. If you're on a device with a hi-res (e.g. Retina) screen, the browser might decide to grab an image with a higher resolution, because it displays at 2x or 3x the number of pixels that are specified in the HTML/CSS. In this situation, even though the largest specified *CSS pixel* size might be 205px, the browser knows it's running on something that has a 3x DPR, so it'll actually grab the 825px image to fill the slot.

Reply:
Does using REMs help with that?